custom/icds_reports/utils/data_pull.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE tmp_daily_attendance AS
+    SELECT
+        awc_id,
+        pse_date,
+        form_location_lat,
+        form_location_long
+        FROM daily_attendance_view WHERE state_id='7fb6f3fe5e7540e7be63c848c28c97ed' AND image_name IS NOT NULL ORDER BY pse_date DESC
+
+CREATE TABLE tmp_daily_attendance_rank AS
+    SELECT
+        awc_id,
+        pse_date,
+        form_location_lat,
+        form_location_long,
+        rank() OVER (
+            PARTITION BY awc_id
+            ORDER BY pse_date DESC
+            )
+
+    FROM tmp_daily_attendance
+
+CREATE TABLE tmp_awc_location_launched AS
```

Comment:
why do you need to create temp table for this?

Reply:
it's not possible to join views and having a join over three table increase the cost of query